lib/shopify_transporter/pipeline/magento/product/top_level_attributes.rb

```diff
@@ -8,10 +8,26 @@ module Magento
       module Product
         class TopLevelAttributes < Pipeline::Stage
           def convert(hash, record)
+            warn_if_too_many_options(hash)
```

Comment:
this would look more ruby if we did something like:
```ruby
warn(hash) if too_many_options?(hash)
```
Sucks that both `warn` and `too_many_options?` require the hash param. Might be nice to make it an instance method so that we could do `warn if too_many_options?`

Reply:
🔥 this is a great suggestion.